EnglishFilter/sections/foreign.txt

```diff
@@ -4789,6 +4789,9 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/111638
+||cloudaaaa.com/a/*.gif
+copytoon251.com##.banner_m
```

Comment:
they change domains from time to time. You can add smth like that: `##a[href^="https://copytoon"][href*="/bannerhit.php?bn_id="]`